.rubocop.yml

```diff
@@ -0,0 +1,62 @@
+Metrics/BlockLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+ 
+Security/Eval:
+  Exclude:
+    - "modules/owner/eval.rb"
+
+Lint/Debugger:
+  Exclude:
+    - "run.rb"
+
+Layout/EndOfLine:
+  EnforcedStyle: lf
+
+Layout/LineLength:
+  Exclude:
+    - "config/custom/custom.sample.rb"
+  Max: 100
+
+Style/GlobalVars:
+  Exclude:
+    - "run.rb"
+
+Security/Open:
+  Exclude:
+    - "modules/helper/download.rb"
+
+# Keep to syntactic linting, instead of complexity.
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/PerceivedComplexity:
+  Enabled: false
+
+# Pending cops for RuboCop 0.80
+Style/HashEachMethods:
+  Enabled: true
+
+Style/HashTransformKeys:
+  Enabled: true
+
+Style/HashTransformValues:
+  Enabled: true
+
+# Pendng cops for RuboCop 0.81
+Lint/RaiseException:
+  Enabled: true
+
+Lint/StructNewOverride:
+  Enabled: true
```

Comment:
Is it worth keeping pending cops manually, or should we enable all? See https://docs.rubocop.org/en/stable/versioning/#pending-cops

Reply:
I don't see a problem with defining them manually?

Reply:
Sounds good - we can continue to manually enable pending cops as they're developed.